src/V3AstNodes.h

```diff
@@ -7107,6 +7107,86 @@ class AstPowUS : public AstNodeBiop {
     virtual int instrCount() const { return widthInstrs() * instrCountMul() * 10; }
     virtual bool signedFlavor() const { return true; }
 };
+class AstPreInc : public AstNodeBiop {
+public:
+    AstPreInc(FileLine* fl, AstNode* lhsp, AstNode* rhsp)
+        : ASTGEN_SUPER(fl, lhsp, rhsp) {
+            dtypeFrom(lhsp);
+        }
+    ASTNODE_NODE_FUNCS(PreInc)
+    virtual AstNode* cloneType(AstNode* lhsp, AstNode* rhsp) { return new AstPreInc(this->fileline(), lhsp, rhsp); }
+    virtual void numberOperate(V3Number& out, const V3Number& lhs, const V3Number& rhs) {
+        out.opAdd(lhs, rhs);
+    }
+    virtual string emitVerilog() { return "%k(++%r)"; }
+    virtual string emitC() { return "VL_SUB_%lq(%lW, %P, %li, %ri)"; } // TODO FIXME
+    virtual string emitSimpleOperator() { return "-"; }
```

Comment:
There is no simple equivalent so:  (Ditto others)

```suggestion
    virtual string emitSimpleOperator() { return ""; }
```